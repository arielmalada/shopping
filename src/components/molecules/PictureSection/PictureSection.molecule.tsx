//#region IMPORTS
import { useContext } from "react";
import { ProductContext } from "../../../contexts/ProductProvider";
//#endregion



//#region MAIN COMPONENTS
const PictureSection: React.FC = () => {
  const productValue = useContext(ProductContext);
  return (
    <img
      src={productValue?.thumbnail}
      alt={productValue?.title || "empty image"}
      className="object-cover w-64 h-64 rounded"
    />
  );
};
//#endregion

export default PictureSection;