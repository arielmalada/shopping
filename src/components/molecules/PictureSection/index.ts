export {default} from './PictureSection.molecule'