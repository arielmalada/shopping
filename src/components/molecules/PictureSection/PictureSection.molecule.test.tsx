import React from 'react';
import renderer from 'react-test-renderer';
import PictureSection from './PictureSection.molecule';

describe('Picture Section', () => {
  it('should be able to render without crashing', () => {
    const tree = renderer.create(<PictureSection />).toJSON();

    expect(tree).toMatchSnapshot();
  });
 
});