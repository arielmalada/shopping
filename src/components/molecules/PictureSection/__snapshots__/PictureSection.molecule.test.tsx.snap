// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout Section should be able to render without crashing 1`] = `
<img
  alt="empty image"
  className="object-cover w-64 h-64 rounded"
/>
`;

exports[`Picture Section should be able to render without crashing 1`] = `
<img
  alt="empty image"
  className="object-cover w-64 h-64 rounded"
/>
`;
